README.md

```diff
@@ -58,20 +22,14 @@ After you're finished please remove all the comments and instructions!
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
+# 📖 [MotoMate] <a name="about-project"></a>
 
-**[your_project__name]** is a...
+- Motomate is a booking app that allows users to easily book and reserve motorcycles online. With Motomate, users can log in and access a wide range of motorcycles to choose from. You can check the back-end of this project [here.](https://github.com/NatanimA/MotoMate-Backend-Api)
```

Comment:
_note: This was also an issue in the backend project._

- [ ] This note:

![image](https://user-images.githubusercontent.com/57421823/223081408-2f08e101-ebc0-4438-9e44-b695e86e2767.png)


This note is just a placeholder to help you add a license file.  After reading it (and following its instructions), kindly remove this note, as it's intended only for you, not your users. 👍